fetch.bs

```diff
@@ -1777,6 +1777,10 @@ specified. [[!CSP]]
 <dfn for=response id=concept-response-range-requested-flag>range-requested flag</dfn>, which is
 initially unset.
 
+<p>A <a for=/>response</a> has an associated
```

Comment:
This should move below the note, below, which relates to the paragraph about range-requested flags, above.

Reply:
ack